<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>790</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>790</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>790</width>
     <height>550</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>790</width>
     <height>550</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <widget class="QTimeEdit" name="timeEditAlarm">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>281</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSetAlarm">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxTimer">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>480</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStartTimer">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>520</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelClock">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>781</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>65</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDeleteAlarm">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>450</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetAlarms">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>281</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEditAlarm">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>281</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPauseTimer">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>440</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonResetTimer">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>440</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTimerDisplay">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>270</y>
      <width>371</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonChangeClockFont">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>250</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>style</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonChangeBackground">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>backround</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonChangeClockColor">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>290</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>color</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
